examples/experimental/dagster-airlift/dagster_airlift/core/sensor/builder.py

```diff
@@ -256,8 +269,38 @@ def build_synthetic_asset_materializations(
         # If there is no dagster_run for this task, it was not proxied.
         # Therefore synthensize a materialization based on the task information.
         if task_id not in dagster_runs_by_task_id:
+            context.log.info(
+                f"Synthesizing materialization for tasks {task_id} in dag {dag_run.dag_id} because no dagster run found."
+            )
             synthetic_mats.extend(
                 materializations_for_task_instance(airflow_data, dag_run, task_instance)
             )
+        else:
+            # We *always* emit for the automapped tasks, even if they are proxied
+            asset_keys_to_emit = automapped_tasks_asset_keys(dag_run, airflow_data, task_instance)
+
+            synthetic_mats.extend(
+                materializations_for_asset_keys_in_task_instance(
+                    dag_run=dag_run, task_instance=task_instance, asset_keys=asset_keys_to_emit
+                )
+            )
```

Comment:
@dpeng817 this is the other big business logic change since the last time you looked.

Reply:
one naming nit but otherwise looks reasonable.